app/assets/javascripts/components/containers/mastodon.jsx

```diff
@@ -56,6 +56,7 @@ import uk from 'react-intl/locale-data/uk';
 import zh from 'react-intl/locale-data/zh';
 import bg from 'react-intl/locale-data/bg';
 import { localeData as zh_hk } from '../locales/zh-hk';
+import bg from 'react-intl/locale-data/ar';
```

Comment:
That should be it, I suppose. However I'm not the author of the commit in question, I think it's @BoFFire .